build.gradle

```diff
@@ -44,8 +46,8 @@ allprojects {
 }
 
 ext {
-    supportLibraryVersion = '28.0.0'
-    butterknifeVersion = '8.6.0'
+    supportLibraryVersion = '1.1.0'
+    butterknifeVersion = '9.0.0-SNAPSHOT'
```

Comment:
Why using snapshot? Latest butterknife supports AndroidX

Reply:
sorry i missed this. updated thanks.